content/docs/composition-vs-inheritance.md

```diff
@@ -45,11 +45,11 @@ function WelcomeDialog() {
 }
 ```
 
-**[Try it on CodePen](https://codepen.io/gaearon/pen/ozqNOV?editors=0010)**
+**[在 CodePen 上尝试](https://codepen.io/gaearon/pen/ozqNOV?editors=0010)**
 
-Anything inside the `<FancyBorder>` JSX tag gets passed into the `FancyBorder` component as a `children` prop. Since `FancyBorder` renders `{props.children}` inside a `<div>`, the passed elements appear in the final output.
+`<FancyBorder>` JSX 标签中的所有内容都会作为一个 `children` prop 传递给 `FancyBorder` 组件。因为 `FancyBorder` 将 `{props.children}` 渲染在一个 `<div>` 中，被传递的这些子组件最终都会出现在输出结果中。
 
-While this is less common, sometimes you might need multiple "holes" in a component. In such cases you may come up with your own convention instead of using `children`:
+有些情况下，你可能需要在一个组件中留出几个“洞”。尽管这种情况相对少见，但是你不需要拘泥于单一的 `children`，还可以自主设计相应的 props 和“洞”的位置来进行组件组合。
```

Comment:
建议：
留出 => 预留出 - 二字词语

有些情况下，……尽管这种情况相对少见，
=>
少数情况下，……在这种情况下
这里不是太强调“情况少见”，而是重点强调“不使用 children”。如果用“尽管”转折，就会忽略了重点。

In such cases you may come up with your own convention instead of using `children`:
这句建议：
这种情况下，我们不使用 `children`，而是需要自行约定：将内容传入一些 props，并从相应的 props 接收。
这里不需要翻译出"洞"或者"插槽"的概念。因为“约定”就能说得很清楚，什么 prop 进去，就从什么 prop 取。